qcodes/dataset/data_set.py

```diff
@@ -567,26 +567,33 @@ def get_data(self,
                  start: Optional[int] = None,
                  end: Optional[int] = None) -> List[List[Any]]:
         """ Returns the values stored in the DataSet for the specified parameters.
-        The values are returned as a list of parallel NumPy arrays, one array
-        per parameter. The data type of each array is based on the data type
-        provided when the DataSet was created. The parameter list may contain
-        a mix of string parameter names, QCoDeS Parameter objects, and
+        The values are returned as a list of lists, SQL rows by SQL columns,
+        e.g. datapoints by parameters. The data type of each element is based on
+        the datatype provided when the DataSet was created. The parameter list may
+        contain a mix of string parameter names, QCoDeS Parameter objects, and
         ParamSpec objects. As long as they have a `name` field. If provided,
         the start and end parameters select a range of results by result count
         (index).
         If the range is empty -- that is, if the end is less than or
         equal to the start, or if start is after the current end of the
         DataSet – then a list of empty arrays is returned.
 
+        For a more type independent and easier to work with view of the data
+        you may want to consider using
+        :py:meth:`qcodes.dataset.data_exporter.get_data_by_id`
```

Comment:
I believe there's a typo here, shouldn't it be `qcodes.dataset.data_export.get_data_by_id`? (no ER after export)